Library/Homebrew/warnings.rb

```diff
@@ -25,8 +26,7 @@ def self.ignore(*warnings)
     end
     return unless block_given?
 
-    result = yield
+    yield
     Warning.clear
-    result
```

Comment:
Wasn't sure if there was any use for this return. At least unused in `brew`. I removed it to simplify typing as this would require arbitrary return (`T.untyped` or something similar).

Reply:
Can you not use `type_parameters` here for this? Fine to make it `void` if it isn't used though.

Reply:
Yes, no use for this return, `void` is fine.